operations/finance/staff-member-expenses/how-to-spend-company-money.md

```diff
@@ -41,7 +41,6 @@ While we're sharing average costs for various items below, you should use your b
 * Laptop stand: Average price **180 USD**
 * Laptop carrying bag: Average price **60 USD**
 * Work-related books: **30 USD** per month or **360 USD** annually
-  * [Recommended resources](https://handbook.mattermost.com/operations/research-and-development/training)
```

Comment:
https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook/professional-development#additional-training-resources